discounts/wasm/merchandise-discount-types/default/schema.graphql

```diff
@@ -0,0 +1,83 @@
+
+   
+schema {
+  query: MerchandiseDiscountRoot
+}
+
+type Address {
+  city: String
+  countryCode: String
+  phone: String
+  poBox: Boolean!
+  provinceCode: String
+  zip: String
+}
+
+type Customer {
+  acceptsMarketing: Boolean
+  email: String
+  id: Int!
+  orderCount: Int
+  phone: String
+  tags: [String!]
+  totalSpent: Money
+}
+
+type DeliveryLine {
+  destination: Address
+  index: Int
+  subscription: Boolean
+}
+
+type MerchandiseDiscountRoot {
+  customer: Customer
+  deliveryLines: [DeliveryLine!]
+  locale: String
+  merchandiseLines: [MerchandiseLine!]
+}
+
+type MerchandiseLine {
+  index: Int
+  price: Money!
+  properties: [Property!]
+  quantity: Int
+  sellingPlan: SellingPlan
+  variant: Variant
+  weight: Int
+}
+
+type Money {
+  currency: String!
+  subunits: UnsignedInt64!
+}
+
+type Product {
+  giftCard: Boolean
+  id: Int!
+  tags: [String!]
+  title: String
+  type: String
+  vendor: String
+}
+
+type Property {
+  key: String!
+  value: String!
+}
+
+type SellingPlan {
+  id: Int!
+}
+
+"""
+Unsigned integer of up to 64 bits, encoded as a msgpack unsinged integer.
```

Comment:
I updated the description since then (and some other things too, I think), but the schemas weren't re-dumped in core. Mind re-dumping the schemas before syncing?